lib/streams/transformations.dart

```diff
@@ -11,3 +11,10 @@ Stream<int> getCashStream(ValueStream<DynamicUserInfo> userInfoStream) =>
 Stream<Int64> getStockPriceStream(
         int stockId, ValueStream<Map<int, Stock>> stockMapStream) =>
     stockMapStream.map((event) => event[stockId]!.currentPrice).distinct();
+
+// returns a stream of stockOwnedMap
+Stream<int> getStockOwnedMapStream(
```

Comment:
```suggestion
Stream<int> getStocksOwnedStream(
```
As it is not returning a map but stocks owned in a single company